app/permissions/concerns/admin_permissions_equity_overrides.rb

```diff
@@ -0,0 +1,15 @@
+require 'active_support/concern'
+
+module AdminPermissionsEquityOverrides
```

Comment:
nit: I don't think this change is related to equity per se, so I suggest dropping that word from the module name.
```suggestion
module AdminPermissionsOverrides
```

Reply:
Good callout. Updated the name :)